tokio-trace/tokio-trace-core/src/dispatcher.rs

```diff
@@ -118,7 +142,8 @@ impl Dispatch {
     /// [`new_span`]: ../subscriber/trait.Subscriber.html#method.new_span
     #[inline]
     pub fn new_span(&self, span: &span::Attributes) -> span::Id {
-        self.subscriber.new_span(span)
+        Self::if_enabled(|| self.subscriber.new_span(span))
```

Comment:
How can we hit this?

Reply:
I guess I'm wondering why this wasn't needed before as it is possible to not have a subscriber.

Reply:
> I guess I'm wondering why this wasn't needed before as it is possible to not have a subscriber.

Sorry, what do you mean? This is to guard against calls to `new_span` inside of the subscriber.

Reply:
Sorry, I meant to delete these comments :) Plz ignore.